google-cloud-spanner-hibernate-performance-testing/src/main/java/com/google/cloud/spanner/hibernate/BenchmarkUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.google.cloud.spanner.hibernate;
+
+import org.apache.log4j.Logger;
+
+public class BenchmarkUtil {
+
+  private static final Logger LOGGER = Logger.getLogger(BenchmarkUtil.class);
+
+  public static void benchmark(Runnable runnable, String description) {
+    LOGGER.info(description);
+    long startMs = System.currentTimeMillis();
+    try {
+      runnable.run();
+    } catch (Exception e) {
+      throw new RuntimeException("Failed to benchmark code: " + description, e);
+    }
+    LOGGER.info("Milliseconds taken: " + (System.currentTimeMillis() - startMs));
```

Comment:
I think it would be more re-usable if you returned the duration, rather than just logging it, and let the caller decide what to do with it.

Reply:
Done.